setup.py

```diff
@@ -57,7 +57,7 @@ def run(self):
 
 
 setup(name='citrine',
-      version='0.49.0',
+      version='0.49.1',
```

Comment:
I figured an improvement (not really a new feature) to alpha functionality didn't count as minor version increment, but I won't argue if someone prefers that.